gsi_wdl_tools/workflow_info.py

```diff
@@ -67,9 +68,28 @@ def outputs(self):
                     description = output_descriptions.get(output_file_name)
             else:
                 raise Exception('Unsupported input type')
+            
+            vidarr_label = []            
+
+            # Check if 'vidarr_label' exists in the output description
+            if 'vidarr_label' in output_descriptions[output.name]:
+                vidarr_label.append(('vidarr_label', output_descriptions[output.name]['vidarr_label']))
+                # If vidarr_label exists, should convert file to file-with-labels
+                if wdl_type == "File":
+                    wdl_type = "Pair[File,Map[String,String]]"
+                elif wdl_type == "Pair[File,Map[String,String]]":
+                    # Extracting existing entries from output.info.expr.right
+                    existing_entries = output.info.expr.right.items
+
+                    # Iterate through the items
+                    for key, value in existing_entries:
+                        key_eval = str(key.eval(None, None))
+                        value_eval = str(value.eval(None, None))          
+                        vidarr_label.append((key_eval, value_eval))
+
             if not description:
                 raise Exception(f"output description is missing for {name}")
-            outputs.append(Output(name=name, wdl_type=wdl_type, description=description))
+            outputs.append(Output(name=name, wdl_type=wdl_type, description=description, vidarr_label=vidarr_label))
```

Comment:
when an output doesn't have a vidarr_label, the column contains `[]`, should it be empty instead?

Reply:
I made the `vidarr_label` data type a list so that there could be multiple of them and so it could be appended to, I don't think I can take out the [] if there are none, is that okay?

Reply:
```suggestion
            outputs.append(Output(name=name, wdl_type=wdl_type, description=description, vidarr_label=vidarr_label if vidarr_label != [] else ""))
```
seem ok?